### DISCUSSION TITLE
clang HelloWorld attempt

### DISCUSSION THREAD

**Discussion by John-L** [termux/termux-app#3717] [Q&A] [answered]
![Screenshot_20231127-204323.png](https://github.com/termux/termux-app/assets/7512058/7ce65fb2-0138-4d9b-8180-68fe9733d362)

Thanks in advance for any clarification you can provide, including possible referrals to other online forums.

**Comment by twaik** [comment, marked as answer]
cpp is not clang, it is "**c** **p**re**p**rocessor". Use `clang++`.

**Comment by John-L** [comment]
I'm dating myself, but I think the Borland C compiler used to be CPlusPlus.
